Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Profile View/NonprofitContainerViewController.m

```diff
@@ -7,23 +7,41 @@
 //
 
 #import "NonprofitContainerViewController.h"
+#import "User.h"
+#import "Nonprofit.h"
 
-@interface NonprofitContainerViewController ()
+@interface NonprofitContainerViewController () <UITableViewDelegate, UITableViewDataSource>
 @property (weak, nonatomic) IBOutlet UIImageView *nonprofitImageView;
 @property (weak, nonatomic) IBOutlet UILabel *nonprofitNameLabel;
-@property (weak, nonatomic) IBOutlet UILabel *nonprofitIsVerifiedLabel;
 @property (weak, nonatomic) IBOutlet UILabel *nonprofitCategoryLabel;
 @property (weak, nonatomic) IBOutlet UILabel *nonprofitWebsiteLabel;
 @property (weak, nonatomic) IBOutlet UILabel *totalDonationValueLabel;
 @property (weak, nonatomic) IBOutlet UILabel *nonprofitDescriptionLabel;
+@property (weak, nonatomic) IBOutlet UIImageView *nonprofitIsVerifiedCheckmark;
+@property (nonatomic, strong) Nonprofit *nonprofit;
+@property (weak, nonatomic) IBOutlet UITableView *foundInBasketsTableView;
 
 @end
 
 @implementation NonprofitContainerViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    // Do any additional setup after loading the view.
+    self.foundInBasketsTableView.delegate = self;
+    self.foundInBasketsTableView.dataSource = self;
+    self.nonprofit = [[User currentUser] nonprofit];
+    if (!self.nonprofit.verificationFiles) { //FIXME: ISVERIFIED
+        self.nonprofitIsVerifiedCheckmark.hidden = YES;
```

Comment:
You can do `self.nonprofitIsVerifiedCheckmark.hidden = !self.nonprofit.verificationFiles` and cover both cases in one line